doc/quick-install.md

```diff
@@ -20,7 +20,17 @@ Then add the same FQDN in the `/etc/hosts` file, associating it with the loopbac
 
 Finally on the same machine test that you can ping the FQDN with: `ping "$(hostname)"`-
 
+
+
+
```

Comment:
sorry that was one to much but i guess that md will be gratefull with me, there was no complete preview in the githup web interface...